auth/login/auth-login.component.ts

```diff
@@ -74,6 +74,8 @@ export class AuthLoginComponent {
         }
     };
 
+    showPrivacyButton = !!environment.zeta.privacyLink;
```

Comment:
As you will need this flag below, you should rather rename this bool to what it stands for. E. g. "privacyLinkDefined"

Reply:
i rename the button see commit